_posts/0000-01-07-ms9700.md

```diff
@@ -6,4 +6,5 @@ title: "Welcome to our slide deck!"
 ![inflatocat](https://octodex.github.com/images/inflatocat.png)
 {: .center}
 
-Caption here
+CAPTION-HERE
```

Comment:
Looks good to me! It looks like you were following the directions [from the manual](https://githubtraining.github.io/training-manual/#/06_working_locally?id=activity-edit-your-file) exactly. 👍